web/tests/search_suggestion_now.test.js

```diff
@@ -127,26 +127,38 @@ test("private_suggestions", () => {
     ];
     assert.deepEqual(suggestions.strings, expected);
 
-    query = "is:private al";
+    query = "is:dm al";
     suggestions = get_suggestions("", query);
     expected = [
-        "is:private al",
-        "is:private is:alerted",
-        "is:private sender:alice@zulip.com",
-        "is:private pm-with:alice@zulip.com",
-        "is:private group-pm-with:alice@zulip.com",
-        "is:private",
+        "is:dm al",
+        "is:dm is:alerted",
+        "is:dm sender:alice@zulip.com",
+        "is:dm pm-with:alice@zulip.com",
+        "is:dm group-pm-with:alice@zulip.com",
+        "is:dm",
     ];
     assert.deepEqual(suggestions.strings, expected);
 
+    // deprecated "is:private" still suggests "is:dm",
+    // but not user suggestions.
```

Comment:
I'm not sure what "user suggestions" means here (and in the other place in this file).

Reply:
Updated to read:

```js
    // "is:private" was renamed to "is:dm", so
    // we suggest "is:dm" to anyone with "is:private"
    // in their muscle memory.
    query = "is:pr";
    suggestions = get_suggestions("", query);
    expected = ["is:dm"];
    assert.deepEqual(suggestions.strings, expected);

    query = "is:private";
    suggestions = get_suggestions("", query);
    expected = ["is:dm"];
    assert.deepEqual(suggestions.strings, expected);
```